### DISCUSSION TITLE
Add support for forwarding messages and attachments

### DISCUSSION THREAD

**Discussion by philjohne** [d99kris/nchat#150] [Ideas]
It would be nice to be able to forward messages to select recipients. As advised in the chat room, I am opening a feature request.

Forwarding a message to 'Saved Messages' is a feature I rely on heavily in the normal Desktop client. It is an easy way to stay organized and I think many people use Telegram this way. But of course I am thinking about forwarding a message to a channel in general. 'Saved Messages' is just a prominent example.

Forwarding messages to other contacts is, at least for me, an integral part of messaging, so I think this feature would increase the overall utility of the command line utility.

**Comment by d99kris** [comment]
Hi @philjohne,

Thanks for suggesting a new feature. I think it's a good idea. I'm currently a bit busy, but I will try to look into it soon.

In any case, I encourage anyone else interested in this feature to upvote it. This will help gauge the level of interest.

**Comment by philjohne** [comment]
Hi Kristofer,

Thanks for looking into it.
And, congratulations to nchat, I really like it :-)

**Comment by wedcanto** [comment]
This is a very, very necessary feature!
Kristofer, it will be very cool if you implement it

**Comment by d99kris** [comment]
Support for forwarding messages has been added in 0eb498f / v5.1.11.
